### PR TITLE
YDA-5808 - Added Base DOI and all the versions of the selected datapackage in VAULT space.

### PR DESCRIPTION
**Pull request by kaur16**
YDA-5808 - Added Base DOI and all the versions of the selected datapackage in VAULT space.

### CODE REVIEW

File: vault/static/vault/css/vault.css [hunk 1]
```diff
@@ -118,3 +118,16 @@ table.dataTable > tbody > tr > td {
     color: #444;
     padding-left:3px;
 }
+
+.meta-content-size {
+    text-align: justify;
```
Comment (lwesterhof) [inline vault/static/vault/css/vault.css]: We can use Bootstrap class `text-justify`

File: vault/static/vault/js/vault.js [hunk 2]
```diff
@@ -790,6 +793,45 @@ function topInformation (dir, showAlert, rebuildFileBrowser = false) {
           $('.alert.is-processing').show()
         } else {
           metadataInfo(dir)
+          if (vaultStatus === 'PUBLISHED') {
```
Comment (lwesterhof) [inline vault/static/vault/js/vault.js]: Also for `DEPUBLISHED`, `PENDING_DEPUBLICATION` and `PENDING_REPUBLICATION`

**Review by lwesterhof — CHANGES_REQUESTED** [review]

File: vault/static/vault/css/vault.css (hunk 1 above)
Comment (kaur16) [reply vault/static/vault/css/vault.css]: Not implementing text justify

**Review by lwesterhof — APPROVED** [review]
LGTM!

**Review by lwesterhof — APPROVED** [review]
LGTM!

**Pull request merged**